app/helpers/about_helper.rb

```diff
@@ -0,0 +1,31 @@
+module AboutHelper
+  def plugins
```

Comment:
if you are creating a controller per action, then I would expect AboutPluginController with only index action.

having said that, I would default to use the API first and only if it doesnt fit create a UI end point

Reply:
I've created About controller namespace, inspired by (this)[http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/] post.

Reply:
in that case why do you need the helper than?
also.. can't we simply use the existing api?